lib/api/core.dart

```diff
@@ -7,6 +7,14 @@ import '../log.dart';
 import '../model/localizations.dart';
 import 'exception.dart';
 
+/// A fused json + utf8 decoder.
+///
+/// This object is an instance of [`_JsonUtf8Decoder`][1] which is
+/// a fast-path present in VM and WASM standard library implementations.
+///
+/// [1]: https://github.com/dart-lang/sdk/blob/6c7452ac1530fe6161023c9b3007764ab26cc96d/sdk/lib/_internal/vm/lib/convert_patch.dart#L55
+final jsonUtf8 = const Utf8Decoder().fuse(const JsonDecoder());
```

Comment:
nit: let's name it so it has a main noun that gives some hint as to what sort of thing this is:
```suggestion
final jsonUtf8Decoder = const Utf8Decoder().fuse(const JsonDecoder());
```

The shorter name indicates that it has something to do with JSON and with UTF-8, but that leaves its real meaning a bit of a mystery — the same name could just as well mean an *encoder*, for example.